manifests/machineconfigdaemon/mcn-guards-validatingadmissionpolicy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "mcn-guards"
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["machineconfiguration.openshift.io"]
+      apiVersions: ["v1alpha1"]
+      operations:  ["CREATE","UPDATE"]
```

Comment:
Q: the daemon seems to also have patch perms. Would this also be needed here?

Reply:
I believe UPDATE encompasses patching as well. VAP only gives you a [smaller "operation" list](https://github.com/djoshy/machine-config-operator/blob/203afde4a93e2c6c7672d6669dd6fe7644ccd388/vendor/k8s.io/api/admissionregistration/v1/types.go#L525) to match on, not sure why :thinking: